pkg/util/mount/mount_linux.go

```diff
@@ -430,3 +430,19 @@ func (mounter *SafeFormatAndMount) getDiskFormat(disk string) (string, error) {
 	// and MD RAID are reported as FSTYPE and caught above).
 	return "unknown data, probably partitions", nil
 }
+
+// IsSamefsGroup is called only for requests to mount an already mounted
+// volume. It checks if fsGroup of new mount request is the same or not.
+// It returns false if it not the same. It also returns current Gid of a path
+// provided for dir variable.
+func IsSamefsGroup(dir string, fsGroup *int64) (bool, int, error) {
```

Comment:
Actually, it seems like the syscall package is deprecated, and instead golang recommends you use the OS-specific packages.  So maybe this needs to be part of the interface, and implemented per OS.